### ISSUE TITLE
Improve startup time

### ISSUE THREAD

**Issue by rugk** [rugk/awesome-emoji-picker#3]
![startup](https://user-images.githubusercontent.com/11966684/55264866-27459180-5276-11e9-93cf-ca4e00c16335.gif)

The GIF does not really convey it, but it does not really start fast. :smiley: 

Actually, it is kinda slow and you can see it first opens only a small white space and then – when the Emoji picker is inserted – it occupies all the space and FF increases the popup size… :smile: 

This should be faster… (no idea, how, though…)

**Comment by Prometheos2** [comment]
### Possible solution?
It might be due to the toolbox charging up everything at the same time instead of doing it incrementally (i.e.: first the toolbox structure, the first rows, then by block of rows).
 - This solution might however returns to the problem of the lags through scrolling (issue _#1_).

**Comment by rugk** [comment]
Hmm… but this is likely a behavior of emoji-mart or preact…

Likely hard to influence… :thinking:

**Comment by rugk** [comment]
Resources are already served locally and I've already optimized emoji-mart in the best way I can think of (https://github.com/rugk/awesome-emoji-picker/issues/8), so I really guess there is nothing I can do here anymore.

Thus I marked this as "help wanted". If anyone has any idea/can analyze what is the issue here, feel free to do so.

In my case, it still takes 1-2 seconds when the popup is opened until I see the whole emoji picker.

**Comment by Prometheos2** [comment]
I have seen that Trello uses a suspiciously similar emoji tool, it seems to start a bit faster, so it might interest you.

**Comment by rugk** [comment]
The problem is likely the startup time of React/preact that needs to init it's own system and start the things.

On a website like Trello, that then likely is already built with React, that is the thing that loads once at website startup and then stays there. The emoji picker "opens" a new website at each start, however…

**Comment by jclsn** [comment]
Yeah, nice Add-on, but this is so annoying... It just makes everything feel sluggish.
